back/tests/WebSpec.hs

```diff
@@ -18,30 +19,79 @@ import Data.Text (Text)
 import qualified Data.Text as T
 -- Files
 import System.Directory
+import System.IO.Temp
 -- URLs
 import Network.URI
 -- Exceptions
 import Control.Monad.Catch
 
 -- Testing
 import Selenium
+import qualified Api
 import qualified Test.WebDriver.Common.Keys as Key
 
 -- Site
 import qualified Guide.Main
-import Guide.Config (Config(..))
+import Guide.Logger
+import Guide.Config (Config(..), def)
+import Text.RE.TDFA.String
 
+-- Spec
+import qualified Test.Hspec as H
+import System.IO
 
 -----------------------------------------------------------------------------
 -- Tests
 -----------------------------------------------------------------------------
 
 tests :: IO ()
-tests = run $ do
-  mainPageTests
-  categoryTests
-  itemTests
-  markdownTests
+tests = withSystemTempFile "test_guide.log" $ \logFile logFileHandle -> do
+  hClose logFileHandle -- It closed for api tests were able to append logs.
```

Comment:
It seems the quoted piece is outdated. Actual version is 
```
  -- Close the log file because otherwise 'run' won't be able to open it
  hClose logFileHandle
```